tasks/tiler/xyz.py

```diff
@@ -61,6 +67,7 @@ def _create_schema_and_table(self, table_schema):
                 mvt_geometry Geometry NOT NULL,
                 geoid VARCHAR NOT NULL,
                 area_ratio NUMERIC,
+                area NUMERIC,
```

Comment:
Why do we need the area now?

Reply:
The MBTiles have the area and I wanted to return the exact same fields that they already have